chia/wallet/puzzles/cat_truths.clib

```diff
@@ -0,0 +1,32 @@
+(
+  (defun-inline cat_truth_data_to_truth_struct (my_id my_parent full_puzhash my_amount innerpuzhash lineage_proof cat_struct)
+    (c
+      (c
+        (c innerpuzhash my_amount)
+        (c lineage_proof cat_struct)
+      )
+      (c
+        (c my_id full_puzhash)
+        (c my_parent ())
```

Comment:
Why bother with the `()` constant node?

Reply:
I see. Future expansion is often a forward-looking good idea. The question is, how would it work here? I guess the idea is CAT inner puzzles that work with the old cat truths would work with new CAT truths too, since they don't know about the new fields anyway (and presumably just ignore the `()` field). Makes sense. In that case, keep the `()`, but put in a comment in so it's clear it's not just an oversight.